htdocs/core/modules/facture/doc/doc_generic_invoice_odt.modules.php

```diff
@@ -420,6 +448,7 @@ public function write_file($object, $outputlangs, $srctemplatepath, $hidedetails
 				$tmparray['object_PREVIOUS_YEAR'] = dol_print_date(dol_time_plus_duree($this->date, -1, 'y'), '%Y');
 				$tmparray['object_YEAR'] = dol_print_date($this->date, '%Y');
 				$tmparray['object_NEXT_YEAR'] = dol_print_date(dol_time_plus_duree($this->date, 1, 'y'), '%Y');
+				$tmparray['object_category_operation'] = $outputlangs->transnoentities("MentionCategoryOfOperations" . $categoryOfOperation);
```

Comment:
To avoid confusion, can you rename the array key from object_category_operation into object_productorservice_operation ? category is already used for tags.